src/dictionaries/words

```diff
@@ -146,6 +148,7 @@ ret
 retrigger
 retriggered
 retriggering
+rmtree
```

Comment:
Manually adding every argument name to the dictionary is going to get painful.

Any ideas on excluding function signatures from spellcheck?

Reply:
@oliver-sanders  I've worked up a proof of concept at https://github.com/cylc/cylc-doc/compare/master...wxtim:cylc-doc:custom_filter_for_Cylc_Words?expand=1 - but I'm not terribly happy with the methods at the top of `cylc.doc.filters` which are scraping the Cylc code.